game-core/src/test/java/games/strategy/engine/random/MartiDiceRollerTest.java

```diff
@@ -13,7 +13,7 @@
 
 public class MartiDiceRollerTest {
   private static final String successMessage =
-      "Some text before<br>your dice are: 2,5,1<p><p>\t</body>\nsome text after";
+      "Some text before<br>your dice are: 22,5,100<p><p>\t</body>\nsome text after";
```

Comment:
Good to see test case updated, though should we perhaps still have the original test data be tested as well for more coverage? Would this be better as a new & additional test rather than updating the existing test?

Reply:
See my explanation. Basically the first number can now have multiple digits as well

Reply:
Again, would it be more comprehensive to have additional test cases rather just this one?
For example, can we have 'zero's', all ones, what if we have just one dice roll, how about more than just three?

Reply:
The original tests passed, yet a 'hotfix' is still needed. By changing the existing test cases, we are losing some data variations. I would suggest that one of the root causes of this problem was insufficient test cases, we did not model in test the actual data set and there was a data set that caused a problem. Generally, bug fixes will receive brand new test cases to reveal the bug before the problem is fixed. That would mean to leave existing test cases along so you can be sure you are not regressing and that you are finding the new problem. As-is, the test coverage seems to be somewhat focused on relatively happy case testing and is lacking potentially interesting test cases.